dodola/core.py

```diff
@@ -28,15 +35,26 @@ def bias_correct_bcsd(
         variable name used in training data
     out_variable : str
         variable name used in downscaled output
+    method : str
+        method to be used in the applied bias correction
     ds_predicted : Dataset
         bias corrected future model data
     """
-
-    # note that time_grouper='daily_nasa-nex' is what runs the
-    # NASA-NEX version of daily BCSD
-    # TO-DO: switch to NASA-NEX version once tests pass
-    model = PointWiseDownscaler(BcsdTemperature(return_anoms=False))
-    model.fit(gcm_training_ds[train_variable], obs_training_ds[train_variable])
-    predicted = model.predict(gcm_predict_ds[train_variable]).load()
+    if method == "BCSD":
+        # note that time_grouper='daily_nasa-nex' is what runs the
+        # NASA-NEX version of daily BCSD
+        # TO-DO: switch to NASA-NEX version once tests pass
```

Comment:
Remove the TO-DO comment?

Reply:
We actually still need to do this so I want to keep it in, but in a different PR in case it somehow breaks things.